pkg/networkmanager/interfaces.js

```diff
@@ -143,7 +143,7 @@ function NetworkManagerModel() {
     var self = this;
     var byteorder = null;
 
-    var client = cockpit.dbus("org.freedesktop.NetworkManager");
+    var client = cockpit.dbus("org.freedesktop.NetworkManager", { superuser: true });
```

Comment:
network-manager in Debian 8 narrowly missed our polkit update, but it is in Debian unstable.  I'll mark this as a hack, but there is no point in filing a bug, I'd say.